### PR TITLE
Added Zhejiang configs

### PR DESCRIPTION
**Pull request by MarekM25**
## Changes

Configs for Zhejiang network (Shanghai public testnet): https://zhejiang.ethpandaops.io/

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

#### Notes on testing

I will sync the network a few times after the Shappella transition and change PR from draft to ready for review
**Update**: synced the network for a few times - all good.

### CODE REVIEW

File: src/Nethermind/Nethermind.Runner/configs/zhejiang.cfg [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "PivotHash": "0x51c3d5bb110bd112b4c5ab47b6b0576d3ac8abb98ba76cce73a6d9f7ebc06b4e",
+    "PivotTotalDifficulty": "1",
+    "FastBlocks": true,
+    "UseGethLimitsInFastBlocks": true,
+    "FastSyncCatchUpHeightDelta": "10000000000"
+  },
+  "EthStats": {
+    "Enabled": false,
+    "Server": "ws://localhost:3000/api",
+    "Name": "Nethermind shandong",
```
Comment (flcl42) [inline src/Nethermind/Nethermind.Runner/configs/zhejiang.cfg:41]: Probably Nethermind zhejiang

Comment (MarekM25) [reply src/Nethermind/Nethermind.Runner/configs/zhejiang.cfg:41]: oh I renamed it somewhere :) thx

Comment (MarekM25) [reply src/Nethermind/Nethermind.Runner/configs/zhejiang.cfg]: ah only in Archive, thx fixed

**Review by flcl42 — APPROVED** [review]
Corresponding launchsettings entries could be cool to add also

**Comment by MarekM25** [comment]
> Corresponding launchsettings entries could be cool to add also

yup, I added them in another branch with one fix :)

**Pull request merged**